Packs/Phishing/ReleaseNotes/3_3_5.md

```diff
@@ -0,0 +1,4 @@
+
+#### Playbooks
+##### Phishing - Generic v3
+- Fixed an issue where a rerun of the playbook would send an email to the reported email address instead of the reporter email address.
```

Comment:
I suggest to add a note here that the playbook should not be re-run because it can cause unexpected behavior. WDYT?

Reply:
Added